_posts/2013-04-02-practices-windows.md

```diff
@@ -39,26 +52,31 @@ Then, to start the automatic configuration, run:
 ember windows
 {% endhighlight %}
 
-##### Node Cli Tool
+[Readmore about this from the Microsoft DX Open Source team](http://felixrieseberg.com/improved-ember-cli-performance-with-windows/)
 
-The automatic configuration tool can also be installed directly, making it
-available in PowerShell & CMD. To install, run:
+#### Enabling symlinks
 
-{% highlight bash %}
-npm install ember-cli-windows -g
-{% endhighlight %}
+In order to create symlinks the account running Ember CLI must have the
+`SeCreateSymbolicLinkPrivilege`. Users in the Administrators group have this
+permission already. However if UAC (User Access Control) is enabled users in
+the Administrators group must run their shell using `Run As Administrator`.
+This is because UAC strips away certain permissions from the Administrators
+group, including `SeCreateSymbolicLinkPrivilege`.
```

Comment:
However, if UAC (User Access Control) is enabled, users in the Administrators group must run their shell using `Run As Administrator` because UAC strips away certain permissions from the Administrators
 +group, including `SeCreateSymbolicLinkPrivilege`.